WtWSMS/common/dynasties/BP_dynasties.txt

```diff
@@ -467,8 +467,8 @@
 	culture =  anglo
 }
 1042170 = {
-	name =  "dynn_Comnurund"
-	culture =  anglo
+	name =  "dynn_Gepanting"
+	culture =  gepid
```

Comment:
I don't understand the link between these two dynasties, why is one replacing the other?

Reply:
Originally Comnurund was used for the gepid family, I don't know why anglo culture was indicated there. About Comnurund, a quick google search turns up AAR from WTWSMS. Gepanting I assigned using the suffix "ing" of the germanic families and the etymology called "Gepanta" mentioned by Iordannes.

Reply:
Ok